### PR TITLE
adding Spark Kotlin demo application...

### PR DESCRIPTION
**Pull request by ludoch**

### CODE REVIEW

File: appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            <goals>
+              <goal>test-compile</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>   
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-war-plugin</artifactId>
+        <version>2.6</version>
```
Comment (lesv) [inline appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml]: 3.1.0 doesn't need the failOnMissingWebXml set.

File: appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        <configuration>
+          <archiveClasses>true</archiveClasses>
+          <failOnMissingWebXml>false</failOnMissingWebXml>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>com.google.appengine</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>1.9.53</version>
```
Comment (lesv) [inline appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml]: 1.9.54 is out.

Comment (ludoch) [inline appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml]: not yet.

File: appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml (hunk 1 above)
Comment (ludoch) [inline appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml]: done

File: appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml (hunk 2 above)
Comment (ludoch) [inline appengine-standard-java8/kotlin-spark-appengine-standard/pom.xml]: not yet.

**Pull request merged**